### DISCUSSION TITLE
Beginner

### DISCUSSION THREAD

**Discussion by mmfarmani** [ImageMagick/ImageMagick#2027] [Help]
{
--
  | use Image::Magick;
  | my $image = Image::Magick->new();
  | $image->BlobToImage(substr($read_data, $pos2, $size));
  | #	$image->Scale('640');
  | $image->Write("$upload_dir/$name");
  | }

The above code, in index.cgi , returns an error message : " Need to upload Magick.pm to /bin folder...

What does it mean? Where should I get Magick.pm ?

Thanks

**Comment by urban-warrior** [comment]
Magick.pm is part of your PerlMagick installation.  We've never seen that error message before.  We would need more details before we could help further.
